package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@zerops/zcli",
+  "version": "v0.0.0-zerops",
```

Comment:
tady teda bude realne v0.16, ale npm to asi nevadi ne?

Reply:
ne